packages/ghc/build.sh

```
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -euo pipefail
+
+# Extract source tarball manually (avoids staging issues with hardlink entries in tarball)
+tar -xof ghc-%{version}-src.tar.xz
```

Shellcheck (0.11.0)

[warning] This { is literal. Check expression (missing ;/\n?) or quote it.

(SC1083)